http/src/main/scala/filodb/http/PrometheusApiRoute.scala

```diff
@@ -41,14 +41,39 @@ class PrometheusApiRoute(nodeCoord: ActorRef, settings: HttpSettings)(implicit a
     // [Range Queries](https://prometheus.io/docs/prometheus/latest/querying/api/#range-queries)
     path( "api" / "v1" / "query_range") {
       get {
-        parameter('query.as[String], 'start.as[Double], 'end.as[Double], 'histogramMap.as[Boolean].?,
-                  'step.as[Int], 'explainOnly.as[Boolean].?, 'verbose.as[Boolean].?, 'spread.as[Int].?)
-        { (query, start, end, histMap, step, explainOnly, verbose, spread) =>
-          val logicalPlan = Parser.queryRangeToLogicalPlan(query, TimeStepParams(start.toLong, step, end.toLong))
+        parameter(
```

Comment:
No real changes to this file right ? I just see whitespace changes. Can be undone ?